e2e/puppeteer_standalone.js

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges UI.
+//
+// Parity Bridges UI is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Parity Bridges UI is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
+const puppeteer = require('puppeteer');
+const SERVER_URL = 'http://localhost:3001';
+
+const chromeOptions = {
+  executablePath:process.env.chrome,
+  headless: false
+};
+
+(async function main() {
+  try {
+    // const pathToExtension = require('path').join(__dirname);
```

Comment:
Please remove this comments.

Reply:
That was note due to "pending" things to be done (if we want this solution and continue testing with it)